src/Components/DetailPage/DetailPage.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState, useEffect } from "react";
+import "./DetailProps";
+import styled from "styled-components";
+import Svg from "../DetailPage/DetailPageSvg";
+import DetailProps from "./DetailProps";
```

Comment:
```suggestion
import React, { useState, useEffect } from "react";
import styled from "styled-components";
import DetailProps from "./DetailProps";
import Svg from "../DetailPage/DetailPageSvg";
```
- import 순서 맞춰주세요
- DetailProps가 두번 import 되고 있네요!

Reply:
수정했습니다!

Reply:
수정하겠습니다!